rust_src/src/buffers.rs

```diff
@@ -199,14 +201,16 @@ impl LispBufferRef {
 
         buffer.into()
     }
+    
     pub fn is_read_only(self) -> bool {
         self.read_only_.into()
     }
-
+    /// Equivalent to BEG macro
```

Comment:
newline here too right?

We have resisted putting direct "this matches blah in C" in remacs because we will have to yank these comments eventually. As we have less and less C they are less and less helpful.